### PR TITLE
Link to related presentations

### PR DESCRIPTION
**Pull request by aradhakrishnanGFDL**
Adding AWS ASDI info webinars and NOAA EDMW slides/video

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -18,3 +18,4 @@ documentation for accessing the cloud data.
    accessing_data
    pangeo_catalog
    licensing_citation
+   presentations
```
Comment (rabernat) [inline docs/index.rst:21]:
```suggestion
   presentation
```

Comment (rabernat) [reply docs/index.rst:21]: This won't work unless you give the document the correct name

Comment (aradhakrishnanGFDL) [reply docs/index.rst]: sure, I updated the typo seeing the preview page, but unable to check if its reflected correctly now?

Comment (aradhakrishnanGFDL) [reply docs/index.rst]: looks ok now https://raw.githack.com/pangeo-data/pangeo-cmip6-cloud/presentations-preview/index.html

**Comment by jbusecke** [comment]
Seems to work for me, Ill merge now.

**Pull request merged**